int/api/websites/all.go

```diff
@@ -60,11 +60,16 @@ func Registry(client *node.Client, candidateDatastoreKeys [][]byte) ([]*models.R
 
 	var metadataKeys []node.DatastoreEntriesKeysAsString
 
+	// Here we need to add the prefix len to the key depending on the type of the key
+	// to get the correct key in the datastore,
+	// each key has a different length and this length is append to the key
 	for _, record := range recordResult {
 		if wallet.AddressChecker(string(record.CandidateValue)) {
-			metadataKeys = append(metadataKeys, node.DatastoreEntriesKeysAsString{
-				Address: string(record.CandidateValue), Key: metaKey,
-			})
+			metadataKey := node.DatastoreEntriesKeysAsString{
+				Address: string(record.CandidateValue),
+				Key:     []byte(metaKey + string(record.CandidateValue)),
+			}
+			metadataKeys = append(metadataKeys, metadataKey)
```

Comment:
I don't understand what you are doing ...

Reply:
The goal is to match metakey="META" + CandidateValue , Which contain the metadata and to serialize this as a byte.
Append this to an array and send this to dataEntries

The commentary on top is an advice to the next dev. He should add the length of the input in prefix of it 
After someone develop the Process to adapt to the type of input

Reply:
> The goal is to match metakey="META" + CandidateValue , Which contain the metadata and to serialize this as a byte. Append this to an array and send this to dataEntries

Why are you doing that ?